src/components/core/DpRegisterFlyout.vue

```diff
@@ -35,6 +35,18 @@ export default {
     DpFlyout
   },
 
+  props: {
```

Comment:
With this component I am again not sure we want to keep it in `demosplan-ui`, as it seems too specific.
If we do want to keep it, we should make it more generic, but since that doesn't apply only to the route props, we can also do it later. As in the other components, I'd suggest to use a single `routes` prop instead of separate props for each route though.

Reply:
Done in:
https://github.com/demos-europe/demosplan-ui/pull/91/commits/2783e7d3955aa18d70de0d0bcaa38d941639aacf
https://github.com/demos-europe/demosplan-project-blp/pull/29/commits/b47c4f3bdb4a09b17705c7e453ed953c17715c86

Reply:
> With this component I am again not sure we want to keep it in demosplan-ui, as it seems too specific.
I completely agree with that. `DpRegisterFlyout` should be moved into `[demosplan-core](https://github.com/demos-europe/demosplan-core/tree/main/client/js/components)/core`. Yes, there is no folder "core" there at the moment but this is a good time to create it. We can also name it "shared", however we want. @salisdemos @hwiem do you agree here?

Reply:
I think it's a good idea, but then for this PR it would suffice to revert the changes to the component imo, as we have a separate ticket for cleaning up the components.

Reply:
what do I have to do here? Revert or leave it now, then move it later?

Reply:
Done in:
https://github.com/demos-europe/demosplan-ui/pull/91/commits/61dec86956bb8021364a51e32d6df0d9f7e672c0
https://github.com/demos-europe/demosplan-project-blp/pull/29/commits/0da46b25babf190bb92529992810511d33e8db2e

Then I would close this PR: https://github.com/demos-europe/demosplan-project-blp/pull/29